include/xtensor/xstrided_view.hpp

```diff
@@ -215,6 +215,8 @@ namespace xt
         const_storage_iterator storage_cbegin() const;
         const_storage_iterator storage_cend() const;
 
+        void set_offset(size_type offset);
```

Comment:
An alternative that allows to remove the trivial implementation of this method:

```cpp
using base_type::set_offset;
```

However, I think it would be better for this method to be in the private section, and to declare `xaxis_iterator` as a friend class.